extensions/dicom-microscopy/src/utils/loadAnnotation.js

```diff
@@ -0,0 +1,71 @@
+import getDicomWebClient from './dicomWebClient';
+
+export default function loadAnnotation({
+  microscopyService,
+  displaySet,
+  extensionManager,
+  servicesManager,
+}) {
+  return new Promise(async (resolve, reject) => {
+    try {
+      const { metadata } = displaySet;
+      console.debug('Loading ANN for display set:', displaySet.metadata);
```

Comment:
remove console debugs please

Reply:
Removed.

Reply:
There is a loglevel library included now that can be used - you can do
const log = ohifLog.getLogger("loadAnnotation");
...
log.debug('Loading ANN for display set...')